packages/commons-test/src/testUtils.ts

```diff
@@ -1271,9 +1271,12 @@ export function createDummyStub<T>(): T {
   return {} as T;
 }
 
-export const getMockNotificationConfig = (): NotificationConfig => ({
-  newEServiceVersionPublished: generateMock(z.boolean()),
-});
+export const getMockNotificationConfig = (): NotificationConfig =>
+  Object.keys(NotificationConfig.shape).reduce((acc, key) => {
```

Comment:
possible alternative (used in the splitters):
```suggestion
  NotificationType.options.reduce((acc, key) => {
```
should be equivalent, maybe yours is more understandable actually

Reply:
oh right... maybe it can be added to packages/readmodel/src/index.ts to export it, anyway not sure it'll be useful